### ISSUE TITLE
Clipboard Feature Flag

### ISSUE THREAD

**Issue by mdewey** [department-of-veterans-affairs/va.gov-team#12402]
Front End
## Product Documentation
[Questionnaires Product Outline ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/product/product-outline.md)
- [Visit Intro Initiative (Proof of Concept)](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/product/initiatives/visit-intro.md)

Create a feature flag in Flipper to hide the clipboard features until we are ready to release this product. 

## Tasks
- [x] Add Feature flag to API
- [x] Disable feature flag in Staging and Production
- [x] Add a feature behind the feature flag in the vets-website
- [ ] Update each feature outline if any technical specifications (ie. API's, Analytics, Implementation details.etc) need documentation
- [ ] Create/work with UT on testing plan
- [x] Add e2e testing
## Acceptance Criteria
- [x] Feature Flag is togglable and configuration in Flipper
- [ ] Front end feature is hidden when the flag is set to false

_Validation = comment in ticket acknowledged review_
- [x] UI review meeting scheduled with product for feature capabilities 
- [x] Validation to mark complete from Product 
- [x] Published to the staging environment, behind a feature flag set to true
- [x] Published to the production environment, behind a feature flag set to false
- [ ] ~~Validation to mark complete from Product and Design they can access with test user~~

**Comment by mdewey** [comment]
Pull Request for API https://github.com/department-of-veterans-affairs/vets-api/pull/4707

**Comment by mdewey** [comment]
API merged into master, waiting for next deployment for staging

**Comment by mdewey** [comment]
Waiting on approval on pull request, tests are passing

**Comment by mdewey** [comment]
Front end is merged (https://github.com/department-of-veterans-affairs/vets-website/pull/13880)

**Comment by mdewey** [comment]
Disabled in production, currently enabled in staging to allow for review

**Comment by mdewey** [comment]
![Screen Shot 2020-08-19 at 2 17 49 PM](https://user-images.githubusercontent.com/1793923/90674195-c467e800-e226-11ea-8804-50283bee349a.png)

Added 2 e2e tests that test that the feature is hidden when the flag is turned off.

**Comment by mdewey** [comment]
![2020-08-19 14-25-56](https://user-images.githubusercontent.com/1793923/90675376-5fad8d00-e228-11ea-95d2-9c31a309fd01.gif)
